Assets/Scripts/Interfaces/ISaveLoad.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface ISaveLoad
+{
+    void SetInt(string dataString, int value);
```

Comment:
dataString isnt a good name - key is probably better

Reply:
Changed to dataID.